src/robusta/core/playbooks/internal/crds.py

```
@@ -0,0 +1,310 @@
+from robusta.core.model.env_vars import KUBECTL_CMD_TIMEOUT_SEC
+from robusta.core.model.base_params import ActionParams
+from robusta.core.playbooks.actions_registry import action
+from robusta.core.model.events import ExecutionBaseEvent
+from robusta.core.reporting.blocks import FileBlock, JsonBlock, TableBlock, MarkdownBlock
+from robusta.core.reporting.base import Finding
+from robusta.utils.error_codes import ActionException, ErrorCodes
+
+
+def run_kubectl_command(command: List[str]) -> str:
+    try:
+        result = subprocess.run(
```

Ruff (0.14.2)

`subprocess` call: check for execution of untrusted input

(S603)